### ISSUE TITLE
/i AJAX Google API is no longer available

### ISSUE THREAD

**Issue by montemarcaldera** [joaoricardo000/whatsapp-bot-seed#12]
http://ajax.googleapis.com/ajax/services/search/web?v=1.0&q=
The API is no longer available :(

**Comment by joaoricardo000** [comment]
?!
http://ajax.googleapis.com/ajax/services/search/web?v=1.0&q=test

Here it is working just fine...

**Comment by montemarcaldera** [comment]
Oops, sorry, wrong link xD
http://ajax.googleapis.com/ajax/services/search/images?v=1.0&q=test

**Comment by joaoricardo000** [comment]
{"responseData": null, "responseDetails": "This API is no longer available.", "responseStatus": 403}

=/
I will look for some open image API now.. Any suggestions?

**Comment by joaoricardo000** [comment]
Text to speech also stopped working.. Not sure if there is a good turnaround for this..

**Comment by montemarcaldera** [comment]
How about Imgur API?

**Comment by joaoricardo000** [comment]
https://github.com/joaoricardo000/whatsapp-bot-seed/pull/21

This pull request uses Bing API to search images.
